onnxruntime/core/providers/cpu/rnn/uni_directional_lstm.cc

```diff
@@ -362,7 +385,8 @@ void UniDirectionalLstm<T>::Compute(const gsl::span<const T>& inputs_arg,
   if (batch_parallel_) {
     double gemm_cost = num_seq_to_compute * hidden_size_x4 * hidden_size_;
     double cost = max_sequence_length * (gemm_cost + num_seq_to_compute);
-    ExecuteLambdaInParallel(sequences_calculator, batch_size_, num_seq_to_compute, cost, thread_pool_);
+    // disable second level parallel expansion
+    ExecuteLambdaInParallel(sequences_calculator, batch_size_, num_seq_to_compute, cost, nullptr);
```

Comment:
This part of the change is not needed now, right?

Reply:
Yes, not related with this bug. will move it to seperate one.

Reply:
Existing logic is correct. Nullptr is set inside the ExecuteLambdaInParallel